mod_reforged/hooks/items/weapons/weapon.nut

```diff
@@ -1,5 +1,17 @@
+::Reforged.HooksMod.hookTree("scripts/items/weapons/weapon", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		if (this.m.Reach < 0)
+		{
+			this.m.Reach += 999 + this.getDefaultReach();
+			this.m.Reach = ::Math.max(0, this.m.Reach);
+		}
+	}
+});
+
 ::Reforged.HooksMod.hook("scripts/items/weapons/weapon", function(q) {
-	q.m.Reach <- 1;
+	q.m.Reach <- -999;
```

Comment:
this could use a comment like 
`// arbitrary negative default value so we preserve simple addition and subtraction operations made by mods`